docs/deployer/container.md

```diff
@@ -1,10 +1,20 @@
 # Container Deployer
 
-The container deployer is a controller that reconciles DeployItems of type `landscaper.gardener.cloud/container`. It executes the given container spec with the injected imports and collect exports from a predefined path.
+The container deployer is a controller that allows you to run whatever coding you need to set up your cloud environment.
+You just need to provide your program as a container image which is then executed by the landscaper in combination
+with the container deployer.
+
+Your image is provided as a reference in a DeployItem of type `landscaper.gardener.cloud/container`. The DeployItem 
+itself must be embedded in a Blueprint and specifies the input data required by your program. The landscaper in
+combination with the container deployer provides these input data during runtime. The output or export data of
+your program must be stored in a predefined file and are collected by the landscaper/container deployer.
```

Comment:
... once your program and thus the container terminates.

Reply:
Hi, I made the requested changes.